test/integration/targets/vmware_guest_find/meta/main.yml

```diff
@@ -0,0 +1,2 @@
+dependencies:
+    - prepare_tests
```

Comment:
I'd leave this out since `prepare_tests` doesn't do anything. It's a carry-over from the old test infrastructure.